packages/devtools_app/lib/src/screens/network/network_model.dart

```diff
@@ -16,31 +14,31 @@ abstract class NetworkRequest with DataSearchStateMixin {
 
   final int _timelineMicrosBase;
 
-  String get method;
+  String? get method;
 
-  String get uri;
+  String? get uri;
 
-  String get contentType;
+  String? get contentType;
 
   String get type;
 
-  Duration get duration;
+  Duration? get duration;
 
-  DateTime get startTimestamp;
+  DateTime? get startTimestamp;
 
-  DateTime get endTimestamp;
+  DateTime? get endTimestamp;
 
-  String get status;
+  String? get status;
 
-  int get port;
+  int? get port;
 
   bool get didFail;
 
   /// True if the request hasn't completed yet.
   bool get inProgress;
 
   String get durationDisplay =>
-      'Duration: ${duration != null ? msText(duration) : 'Pending'}';
+      'Duration: ${duration != null ? msText(duration!) : 'Pending'}';
```

Comment:
remove the `=>` and use brackets instead so that we can hoist duration into a local var and remove the double null assert